diff --git a/node_modules/rc-menu/es/DOMWrap.js b/node_modules/rc-menu/es/DOMWrap.js
index ff77397..0ec1378 100644
--- a/node_modules/rc-menu/es/DOMWrap.js
+++ b/node_modules/rc-menu/es/DOMWrap.js
@@ -45,7 +45,7 @@ var FLOAT_PRECISION_ADJUST = 0.5; // Fix ssr
 
 if (canUseDOM) {
   // eslint-disable-next-line global-require
-  require('mutationobserver-shim');
+// not needed in electron  require('mutationobserver-shim');
 }
 
 var DOMWrap =
